src/vivarium_census_prl_synth_pop/results_processing/names.py

```diff
@@ -19,6 +19,7 @@ def get_given_name_map(
     artifact: Artifact,
     randomness: RandomnessStream,
     seed: str,
+    _: List[int],
```

Comment:
I'd recommend doing this for all of the mapper functions (other than ssn and itin obviously). Then you don't have to pass the `all_seeds` argument in `get_middle_initial_map`

```suggestion
    *_: Any,
```

Reply:
lol well that's much nicer than my ["dummy_list"]s